src/nodes/create_trimesh_collider.gd

```diff
@@ -0,0 +1,20 @@
+tool
+extends ConceptNode
+
+
+func _init() -> void:
+	unique_id = "create_trimesh_collider"
+	display_name = "Create collision trimesh"
+	category = "Meshes"
+	description = "Creates a trimesh collider from a mesh"
+
+	set_input(0, "Node", ConceptGraphDataType.MESH)
+		
+	set_output(0, "", ConceptGraphDataType.MESH)
+	
+func _generate_outputs() -> void:
+	var mesh:MeshInstance = get_input_single(0)
+	
```

Comment:
Better add a check here just in case the mesh object isn't really a MeshInstance